content/agent/exporters/prometheus/_index.md

```diff
@@ -0,0 +1,129 @@
+---
+title: "Prometheus"
+date: 2019-02-13T16:14:17-08:00
+logo: /img/prometheus-logo.png
+---
+
+- [Introduction](#introduction)
+- [Configuration](#configuration)
+    - [Format](#format)
+- [End to end example](#end-to-end-example)
+    - [Running ocagent](#running-ocagent)
+    - [Running Prometheus](#running-prometheus)
+    - [Running Application code](#running-application-code)
+- [Results](#results)
+    - [All metrics](#all-metrics)
+    - [Rates for individual line length buckets](#rates-for-individual-line-length-buckets)
+    - [p99th latencies](#p99th-latencies)
+    - [Rates for line counts](#rates-for-line-counts)
+- [References](#references)
+
+### Introduction
+
+ocagent allows you to export metrics that are collected from client libraries to Prometheus.
+
+### Configuration
+
+In the ocagent's YAML configuration file, under section "exporters" and sub-section "prometheus"
+
+#### Format
+With these fields below
+
+```yaml
+exporters:
+  prometheus:
+    namespace: "<namespace>"
+    address: "<port:host>"
+    const_labels: {
+        "<key1>":"<value1>",
+        "<key2>":"<value2>"
+    }
+```
+
+### End to end example
+
+In this end-to-end example, we'll have ocagent running and a couple of Go applications
+that use the [Go ocagent-exporter](/exporters/supported-exporters/go/ocagent)
+to send over metrics and traces to ocagent.
+
+![](/images/ocagent-exporter-prometheus-architecture-schematic.png)
+
+### Running ocagent
+
+On starting ocagent with the configuration below:
+```yaml
+# Saved in oca.yaml
+exporters:
+  prometheus:
+    namespace: "promdemo"
+    address: "localhost:9999"
+    const_labels: {
+        "vendor": "otc"
+    }
+
+receivers:
+  opencensus:
+    address: "localhost:55678"
+```
+
+On running ocagent:
+
+```shell
+./bin/ocagent --config oca.yaml
+```
+
+### Running Prometheus
+As specified above in [Running ocagent with exporter](#running-ocagent-with-exporter) by "address": "localhost:9090", please have your Prometheus server running by also configuring
+its configuration file `prom.yaml`
+```yaml
+# Saved into prom.yaml
+scrape_configs:
+  - job_name: 'agent1'
+    scrape_interval: 5s
+    static_configs:
+      - targets: ['localhost:8888']
```

Comment:
Port should be 9999: https://github.com/census-instrumentation/opencensus-website/pull/586/files#diff-54accb170b03014fc8afecb0aede91b2R59.

Reply:
In that case I'll also update above to 8888, thanks for the catch!